app/scripts/views/sign_in.js

```diff
@@ -310,4 +325,11 @@ function (Cocktail, p, BaseView, FormView, SignInTemplate, Session,
   );
 
   return View;
+
+  function interTabSignInHandler (event) {
```

Comment:
Another edge case is tab 1 is at `/signin`, in tab 2, the user signs up & verifies, tab 1 just sits there. Am I too bothered? Not really.